src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs

```diff
@@ -42,7 +42,7 @@ public class ApplicationController : Controller
             ISiteService siteService,
             IStringLocalizer<ApplicationController> stringLocalizer,
             IAuthorizationService authorizationService,
-            IRoleProvider roleProvider,
+            IRoleService roleService,
```

Comment:
@domonkosgabor any chance you could also resolve this dependency in an optional way, like you did for the main OpenID controller? Thanks!

Reply:
Sure. What do you think, what should we do if there aren't any service available that implements `IRoleService` when creating/editing an application?
Just simply show a warning message in this page to the user about there will be no RoleEntrys in the list?